todo_app/app.py

```diff
@@ -1,15 +1,29 @@
-from flask import Flask
-
+from types import GetSetDescriptorType
```

Comment:
This (and a couple of other imports; `Request`, `escape`, `_DEFAULT_ITEMS`) look to be unused, so we should clear those out